code/datums/traits/good.dm

```diff
@@ -96,6 +96,33 @@
 	)
 	H.equip_in_one_of_slots(B, slots , qdel_on_fail = TRUE)
 
+/datum/quirk/multilingual
+	name = "Multilingual"
+	desc = "You spent a portion of your life learning to understand an additional language. You may or may not be able to speak it based on your anatomy."
+	value = 2
+	mob_trait = TRAIT_MULTILINGUAL
+	gain_text = "<span class='notice'>You have learned to understand an additional language.</span>"
+	lose_text = "<span class='danger'>You have forgotten how to understand a language.</span>"
+
+/datum/quirk/multilingual/on_spawn()
+	var/mob/living/carbon/human/H = quirk_holder
+	var/datum/language_holder/LH
+
+	var/datum/language/newLang
+	if(H.job != "Curator")
+		LH.add_blocked_language(/datum/language/narsie, LANGUAGE_ATOM)
```

Comment:
Isn't this just going to null.add_blocked_language(/datum/language/narsie, LANGUAGE_ATOM) and runtime and not work at all, or am I misunderstanding how add_blocked_language works?

Reply:
LH is effectively a library of nonantag languages from which a random one is pulled on spawn. Since H has no languages blocked and is not copying the language setup of LH, languages like Narsian, Codespeak, etc. can still be added when the conditions fit.

Reply:
Yeah but you never initialize LH